component/otelcol/receiver/jaeger/jaeger.go

```diff
@@ -30,8 +29,7 @@ func init() {
 
 // Arguments configures the otelcol.receiver.jaeger component.
 type Arguments struct {
-	Protocols      ProtocolsArguments       `river:"protocols,block"`
-	RemoteSampling *RemoteSamplingArguments `river:"remote_sampling,block,optional"`
```

Comment:
This is a breaking change, right? It should be documented as such in the changelog.

Reply:
There was a [discussion](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/24186) on this in the upstream PR, and I'm going to backtrack and just add an entry in the upgrade guide and in the changelog 😅